server/src/api/api.graphql

```diff
@@ -78,3 +84,54 @@ type Category {
     # The human-readable, user-facing name for this category
     category_name: String!
 }
+
+enum RequestStatus {
+	SUBMITTED
```

Comment:
could potentially support waitlist here too :) an issue for later

Reply:
It's probably easiest to add now, but IMO waitlist is mostly a frontend display trick since the Submitted status is basically the same thing (waiting for approval and requests don't get approved unless they can be fulfilled)